packages/realm/bindgen/src/templates/jsi.ts

```diff
@@ -486,7 +486,6 @@ function convertToJsi(addon: JsiAddon, type: Type, expr: string): string {
           assert.fail("Should never see AsyncResult here");
       }
       assert.fail(`unknown template ${type.name}`);
-      break;
```

Comment:
Do you want to add a `/* eslint-disable-next-line no-fallthrough */` for the removed `break` statements? Or if possible, I think it'd read better if keeping the `break` statements but instead silence the `Unreachable code detected`. (Similarly for the Node template.) What do you think?

Reply:
I have no idea why this change got applied 👀

Reply:
It seems an update to TypeScript recognize the `break` as undead code and removes it in my editor on save, because of the `never` return-type from `assert.fail` 🤔

Reply:
I think [this comment](https://github.com/realm/realm-js/pull/6598#discussion_r1558163311) was meant for this thread 🙂 Good solution 👍